Packs/CiscoUmbrellaReporting/Integrations/CiscoUmbrellaReporting/README.md

```diff
@@ -328,93 +203,47 @@ List of identities ordered by the number of requests made matching the categorie
 | UmbrellaReporting.Identity.bandwidth | Number | The total bandwidth of proxy requests uploaded and downloaded for this identity. | 
 | UmbrellaReporting.Identity.rank | Number | The rank of the result based on the number of requests. | 
 | UmbrellaReporting.Identity.counts.allowedrequests | Number | Number of requests that were allowed. | 
-| UmbrellaReporting.Identity.counts.blockedrequests | Number | Number of requests that were blocked. | 
-| UmbrellaReporting.Identity.counts.requests | Number | Total number of requests. | 
-| UmbrellaReporting.Identity.identity.id | Number | Identity ID. | 
-| UmbrellaReporting.Identity.identity.type.id | Number | Origin type for the identity. | 
-| UmbrellaReporting.Identity.identity.type.type | String | Origin type name for the identity. | 
-| UmbrellaReporting.Identity.identity.type.label | String | Origin type label for the identity. | 
-| UmbrellaReporting.Identity.identity.label | String | Label for the identity. | 
-| UmbrellaReporting.Identity.identity.deleted | Boolean | Indicates whether the identity was deleted. | 
-
-#### Command example
-```!umbrella-reporting-identity-list limit=2```
-#### Context Example
-```json
-{
-    "UmbrellaReporting": {
-        "Identity": [
-            {
-                "bandwidth": 7974662,
-                "counts": {
-                    "allowedrequests": 29540,
-                    "blockedrequests": 72,
-                    "requests": 29753
-                },
-                "identity": {
-                    "deleted": false,
-                    "id": 589064228,
-                    "label": "DESKTOP-IIQVPJ7",
-                    "type": {
-                        "id": 9,
-                        "label": "Roaming Computers",
-                        "type": "roaming"
-                    }
-                },
-                "rank": 1,
-                "requests": 29753
-            },
-            {
-                "bandwidth": null,
-                "counts": {
-                    "allowedrequests": 17950,
-                    "blockedrequests": 59,
-                    "requests": 18082
-                },
-                "identity": {
-                    "deleted": false,
-                    "id": 593805843,
-                    "label": "S\u2019s MacBook Pro",
-                    "type": {
-                        "id": 9,
-                        "label": "Roaming Computers",
-                        "type": "roaming"
-                    }
-                },
-                "rank": 2,
-                "requests": 18082
-            }
-        ]
-    }
-}
-```
-
-#### Human Readable Output
-
->### Identities List
->|Identity|Requests|
->|---|---|
->| DESKTOP-IIQVPJ7 | 29753 |
->| S’s MacBook Pro | 18082 |
-
-
-### umbrella-reporting-event-type-list
+### umbrella-reporting-category-list
```

Comment:
the `umbrella-reporting-event-type-list` command still exists - why removing its documentation?